127_1_uzdavinysAbecele.cpp

```diff
@@ -0,0 +1,66 @@
+const int MAXZODZIU = ...;
+
+int sk;                             // žodžių skaičius
+string z[MAXZODZIU];                // žodžių masyvas
+int n;                              // viršūnių skaičius
+map<char, bool> virsune;            // ar grafe yra raidę atitinkanti viršūnė
+map<pair<char, char>, bool> lankas;
+map<char, int> ieinLanku;
+
+void sudarykGrafa () {
+    // išvalomos duomenų struktūros
+    n = 0;
+    virsune.clear();
+    lankas.clear();
+    ieinLanku.clear();
+
+    // sudaromas grafas
+    zodziai[sk] = ""; // pridedame tuščią žodį
+    for (int i = 0; i < sk; i++) {
+        for (int j = 0; j < (int)z[i].size(); j++) {
+            // jei randama naujų raidžių - jos įtraukiamos į grafą
+            if (!virsune[z[i][j]]) {
+                n++;
+                virsune[z[i][j]] = true;
+            }
+
+            int m = min((int)z[i].size(), (int)z[i+1].size());

```

Comment:
`i+1` užeina už rėžių ciklo gale

Reply:
Kaip supratau, būtent dėl to pridedamas tas tuščias žodis:
`z[sk] = ""`

Reply:
Ai jo, tai tada masyvo dydį reiktų padaryt `MAXZODZIU+1`